op-node/rollup/derive/engine_queue.go

```diff
@@ -684,6 +707,61 @@ func (eq *EngineQueue) StartPayload(ctx context.Context, parent eth.L2BlockRef,
 	return eq.ec.StartPayload(ctx, parent, attrs, updateSafe)
 }
 
+// tryBackupUnsafeReorg tries to reorg(restore) unsafe head to backupUnsafeHead.
+func (eq *EngineQueue) tryBackupUnsafeReorg(ctx context.Context) error {
+	// Only try once because execution engine may forgot backupUnsafeHead
+	// or backupUnsafeHead is not part of the chain.
+	// Exception: Retry when forkChoiceUpdate returns non-input error.
+	eq.checkBackupUnsafeReorg = false
+	// This method must be never called when EL sync. If EL sync is in progress, early return.
+	if eq.ec.UnsafeL2Head().Hash != eq.ec.EngineSyncTarget().Hash {
+		eq.log.Warn("Attempting to update forkchoice state while engine is P2P syncing.")
```

Comment:
```suggestion
		eq.log.Warn("Attempting to update forkchoice state while EL sync.")
```
just for consistency